File: BankingTool.Model/Dto/BankAccount/GetTransactionsListDto.cs
namespace BankingTool.Model.Dto.BankAccount
{
    public class GetTransactionsListDto
    {
        public List<TransactionsListDto> TransactionsList { get; set; }
        public TransactionsListAccountInfoDto AccountInfo { get; set; }
        public List<TransactionsListCardInfoDto> CardInfo { get; set; }
    }
}

File: bankingtool.client/src/app/dto/get-transactions-list-dto.ts
import { TransactionsListAccountInfoDto } from "./transactions-list-account-info-dto";
import { TransactionsListCardInfoDto } from "./transactions-list-card-info-dto";
import { TransactionsListDto } from "./transactions-list-dto";

export interface GetTransactionsListDto {
  transactionsList: TransactionsListDto[];
  accountInfo: TransactionsListAccountInfoDto;
  cardInfo: TransactionsListCardInfoDto[];
}
